xarray/tests/test_indexing.py

```diff
@@ -6,12 +6,85 @@
 
 from xarray import Dataset, DataArray, Variable
 from xarray.core import indexing, utils
+from xarray.core.pycompat import integer_types
 from . import TestCase, ReturnItem
 
 
+def canonicalize_indexer(key, ndim):
```

Comment:
If we don't need these functions anymore, please delete them! They'll always be in git history (or prior releases) if we need them later.